### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by vhavlena**

### CODE REVIEW

**Review by jurajsic — APPROVED** [review]
Looks good

File: README-Starexec.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+make install
+```
+3. Install Mata
+```shell
+git clone 'https://github.com/VeriFIT/mata.git'
+cd mata
+vim CMakeLists.txt
+# ... comment out the following two lines:
+#       include(CodeCoverage)
+#       setup_target_for_coverage(${PROJECT_NAME}_coverage tests coverage)
```
Comment (Adda0) [inline README-Starexec.md:24]: This should not be necessary. If it is, we did something wrong in Mata CMake.

File: README-Starexec.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+```
+4. Compile Noodler:
+
+```shell
+git clone 'https://github.com/VeriFIT/z3-noodler.git'
+mkdir z3-noodler/build
+cd z3-noodler/build
+vim ../src/test/CMakeLists.txt
+# ... comment out the following line:
+#       add_subdirectory(noodler)
```
Comment (Adda0) [inline README-Starexec.md:36]: We should make compiling Catch2 tests optional, as in Mata. At least in the future.

**Review by Adda0 — APPROVED** [review]
LGTM.

File: README-Starexec.md (hunk 1 above)
Comment (vhavlena) [reply README-Starexec.md:24]: True. The problem is that the Starexec readme is obsolete and I don't have time to run it inside starexec vm again.

File: README-Starexec.md (hunk 2 above)
Comment (vhavlena) [reply README-Starexec.md:36]: The same problem as above. As soon as someone will have time, we can update it.

Comment (Adda0) [reply README-Starexec.md:36]: I will make an issue for this, and we will fix it later.

File: README-Starexec.md (hunk 1 above)
Comment (Adda0) [reply README-Starexec.md:24]: I will make an issue for this, and we will fix it later.

Comment (jurajsic) [reply README-Starexec.md:24]: I do not think there is any point in updating readme for starexec, it was only for smt-comp, and they are supposed to move away from starexec next year.

Comment (vhavlena) [reply README-Starexec.md:24]: Ok, lets keep it as it is

File: README.md [hunk 3]
```diff
@@ -3,9 +3,7 @@
 Z3-Noodler is an SMT solver for string constraints such as those that occur in symbolic execution and analysis of programs, 
 reasoning about configuration files of cloud services and smart contracts, etc.
 Z3-Noodler is based on the SMT solver [Z3](https://github.com/Z3Prover/z3), in which it replaces the solver for the theory of strings. 
-The core of the string solver relies on the equation stabilization algorithm from the paper
-
-> F. Blahoudek, Y. Chen, D. Chocholatý, V. Havlena, L. Holík, O. Lengál, and J. Síč. [Word Equations in Synergy with Regular Constraints](https://link.springer.com/chapter/10.1007/978-3-031-27481-7_23).  In *Proc. of FM’23*, Lübeck, Germany, volume 14000 of LNCS, pages 403–423, 2023. Springer-Verlag.
+The core of the string solver implements several decision procedures, but mainly it relies on the equation stabilization algorithm.
```
[the remark this answers is not in the document]
Comment (vhavlena) [reply README.md]: added

**Comment by jurajsic** [comment]
Looks good, I think we can merge.

**Pull request merged**